Application/Test.Common/MsSqlDatabaseHelper.cs

```diff
@@ -38,6 +41,49 @@ public void CopyDatabase(string originalDatabaseName, string newDatabaseName)
             }
         }
 
+        public string DownloadAndRestoreDatabase()
+        {
+            try
+            {
+                var psScript = @"
+                    $ErrorActionPreference = 'Stop'
+
+                    .\Initialize-PowershellForDevelopment.ps1
+
+                    $settings = @{ 
+                        ApiSettings = @{ Engine = 'SQLServer' } 
+                        ConnectionStrings = @{ EdFi_Ods = '" + _odsConnectionString + @"' } 
+                    }
+                    Set-DeploymentSettings $settings
+
+                    Reset-TestPopulatedTemplateDatabase
+                ";
+
+                var info = new ProcessStartInfo("powershell", psScript)
+                {
+                    WorkingDirectory = Path.GetFullPath("../../../../../../Ed-Fi-ODS-Implementation"),
+                    RedirectStandardError = true,
+                    RedirectStandardOutput = true,
+                };
+
+                using var process = Process.Start(info);
+
+                process.OutputDataReceived += (sender, e) => Console.WriteLine(e.Data);
```

Comment:
Console.WriteLine doesn't seem to work in tests. A more robust approach would be to use Log4net like EdFi.Ods.Utilities.Migration.Tests does.

For now I guess we can remove these events (together with the `RedirectStandardError = true, RedirectStandardOutput = true` and `process.BeginOutputReadLine(); process.BeginErrorReadLine();` )